backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/action/MoveOrCopyImageGroupParameters.java

```diff
@@ -159,8 +159,16 @@ public Guid getSourceDomainId() {
         return sourceDomainId;
     }
 
-    public void setSourceDomainId(Guid value) {
-        sourceDomainId = value;
+    public void setSourceDomainId(Guid sourceDomainId) {
+        this.sourceDomainId = sourceDomainId;
+    }
+
+    public Guid getDestDomainId() {
```

Comment:
@barpavel missing JsonIgnore here, no?

Reply:
I believe it's enough to specify it on either the setter or getter

Reply:
In that case, it's done :)
Resolving the comment.